lib/conjur/debify.rb

```diff
@@ -196,6 +196,17 @@ def ignore_file? f
   end
 end
 
+def copy_packages_from_container(container, package_name, dev_package_name)
+  Conjur::Debify::Utils.copy_from_container container, "/src/#{package_name}"
+  puts "#{package_name}"
+  begin
+    Conjur::Debify::Utils.copy_from_container container, "/dev-pkg/#{dev_package_name}"
```

Comment:
i don't think that this simplifies the code but adds complexity

Reply:
i don't think that this will simplify the code. the current one is good and will be called once for each file type. i don't expect that we'll need the extra generic level and adding it will add redundant complexity.